kubernetes/README.md

```diff
@@ -138,7 +138,7 @@ And then use the secret with the containers. For the init container, use the cmd
 ```
       initContainers:
       - name: init
-        image: jasperserver-pro-cmdline:k8s-7.5.0
+        image: jasperserver-pro-cmdline:k8s-7.8``.0
```

Comment:
I'm assuming this is a typo, I will clean it up.

Reply:
Oops, was reading the commits in the wrong direction.. 😸  I think this is covered later.